CMakeLists.txt

```diff
@@ -163,3 +163,55 @@ INSTALL(FILES ${CMAKE_BINARY_DIR}/include/singa/singa_config.h DESTINATION
 INSTALL (DIRECTORY ${CMAKE_BINARY_DIR}/lib DESTINATION
     ${CMAKE_INSTALL_PREFIX}
     PATTERN "*libgtest.a" EXCLUDE )
+
+IF (USE_PYTHON)
+INSTALL(DIRECTORY ${PROJECT_SOURCE_DIR}/python 
```

Comment:
It think python setup.py install can copy these files. Then we don't need to copy them explicitly.
https://github.com/nudles/incubator-singa/blob/conda/CMakeLists.txt#L166